script/make_release.py

```diff
@@ -0,0 +1,219 @@
+import subprocess
+import threading
+import os
+import shutil
+import uuid
+import stat
+import concurrent.futures
+import argparse
+import logging
+import platform
+
+DEFAULT_ARCHS = [
+    'x86-64-vnni512',
+    'x86-64-vnni256',
+    'x86-64-avx512',
+    'x86-64-bmi2',
+    'x86-64-avx2',
+    'x86-64-sse41-popcnt',
+    'x86-64-modern',
+    'x86-64-ssse3',
+    'x86-64-sse3-popcnt',
+    'x86-64',
+    'x86-32-sse41-popcnt',
+    'x86-32-sse2',
+    'x86-32',
+    'general-64',
+    'general-32'
+]
+
+parser = argparse.ArgumentParser(description='Stockfish release maker', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--sde', default=None, type=str,
+                    help='Path to Intel SDE. If not specified it will not be used. If specified it will be used as a fallback for architectures that fail to compile.')
```

Comment:
This little change is a life changer for a SDE nooby like me :)
`help='Absolute path to Intel SDE application.`